src/applications/mhv/secure-messaging/sass/folder-thread-list-view.scss

```diff
@@ -6,7 +6,14 @@
 .thread-list-draft {
   color: $color-secondary-darkest;
   font-weight: 700;
+
+}
+
+.reply-draft-label {
+  white-space: nowrap;
+  padding-right: 3px;
```

Comment:
based on the into earlier today, would you like to use the VA DST class for padding on an element itself instead?